examples/QIR/Optimization/Hello/Hello.csproj

```diff
@@ -8,8 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.25.222597-alpha" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.25.222597" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.25.228311" GeneratePathProperty="true" />
```

Comment:
```suggestion
    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.25.228311-alpha" GeneratePathProperty="true" />
```

Reply:
Thanks for the catch, @kuzminrobin !

Definitely a must. I'll commit the suggestion. FYI, @DmitryVasilevsky